opte/src/engine/port.rs

```diff
@@ -1193,17 +1194,25 @@ impl Port {
                 }
 
             } else if #[cfg(feature = "usdt")] {
-                let ifid_s = ifid.to_string();
+                let flow_b_s = ifid_b.to_string();
+                let flow_a_s = ifid_a.to_string();
                 let res_str = match res {
                     Ok(v) => format!("{:?}", v),
                     Err(e) => format!("ERROR: {:?}", e),
                 };
 
                 crate::opte_provider::port__process__return!(
-                    || (dir, &self.name, ifid_s, epoch, pkt.mblk_addr(), res_str)
+                    || (
+                        (dir, self.name.as_str()),
+                        (flow_b_s.as_ref(), flow_a_s.as_ref()),
+                        epoch,
+                        pkt.mblk_addr(),
+                        res_str
+                    )
                 );
             } else {
-                let (_, _, _, _, _) = (dir, ifid, epoch, pkt, res);
+                let (_, _, _, _, _, _) =
```

Comment:
I learned recently that you can use `(..)` here to collect all remaining items. Kinda cool.

Reply:
Yea this is nice. I'll start changing places as I come across  them.